bin/propolis-server/src/lib/migrate/mod.rs

```diff
@@ -57,10 +57,12 @@ pub enum MigrateRole {
     Destination,
 }
 
+#[derive(Debug)]
```

Comment:
do you mind fixing up scripts/live-migration-times.md to reflect the new phases?

Reply:
Done in 0a387ff, 8bbd69d, and 071df1e (took a few tries to get everything arranged in a way I was happy with).

I thought briefly that I would want to have pause/resume probes in the VM state driver to help with this, so I added those in e11dacc. I didn't end up using the probes, but the other cleanup in that commit seems useful to me, so I kept it in this sequence.